instrumentation-core/spring/spring-webflux-5.0/src/main/java/io/opentelemetry/instrumentation/spring/webflux/client/SpringWebfluxHttpClientTracer.java

```diff
@@ -71,4 +68,18 @@ protected String responseHeader(ClientResponse clientResponse, String name) {
     List<String> headers = clientResponse.headers().header(name);
     return !headers.isEmpty() ? headers.get(0) : null;
   }
+
+  @Override
+  protected Setter<ClientRequest> getSetter() {
+    return null;
+  }
+
+  @Override
+  protected String getInstrumentationName() {
+    return "io.opentelemetry.auto.spring-webflux-5.0";
+  }
+
+  public Tracer getTracer() {
+    return tracer;
+  }
```

Comment:
we will get rid of this soon, you can accept https://github.com/heyams/opentelemetry-java-instrumentation/pull/1, or i can send that in as separate PR after your PR